File: be/src/main/java/com/project/popupmarket/dto/payment/ReceiptsInfoTO.java
package com.project.popupmarket.dto.payment;

import lombok.Builder;
import lombok.Getter;

import java.math.BigDecimal;
import java.time.LocalDate;

@Getter
@Builder
public class ReceiptsInfoTO {
    private String orderId;
    private String status;
    private String landTitle;
    private String customerName;
    private BigDecimal price;
    private BigDecimal amount;
    private LocalDate start;
    private LocalDate end;
}

File: be/src/main/java/com/project/popupmarket/dto/payment/ReceiptsInfoTO.kt
package com.project.popupmarket.dto.payment

import java.math.BigDecimal
import java.time.LocalDate

data class ReceiptsInfoTO(
    val orderId: String,
    val status: String,
    val landTitle: String,
    val customerName: String,
    val price: BigDecimal,
    val amount: BigDecimal,
    val start: LocalDate,
    val end: LocalDate
)